src/Neo/Persistence/ISnapshot.cs

```diff
@@ -35,5 +37,40 @@ public interface ISnapshot : IDisposable, IReadOnlyStore
         /// <param name="key">The key of the entry.</param>
         /// <param name="value">The data of the entry.</param>
         void Put(byte[] key, byte[] value);
+
+        /// <summary>
+        /// Load data from json
+        ///
+        /// Expected data (in base64):
+        /// 
+        /// - "key":"value"
+        /// - "prefix": { "key":"value" }
+        /// </summary>
+        /// <param name="json">Json Object</param>
+        public void LoadFromJson(JObject json)
```

Comment:
Why is this in this `Interface`? It is an `Interface` dotnet standard doesn't allow this.

Reply:
We can move the interface to a base class or an extension method